src/styles/main.scss

```diff
@@ -1,3 +1,25 @@
-body {
+@import "./utils/variables";
+@import "./blocks/header.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
+
+* {
+  box-sizing: border-box;
   margin: 0;
 }
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+a {
```

Comment:
Setting `display: block` for all anchor tags may not be suitable for all cases. Consider applying this property only when needed by class name.